vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -116,82 +115,16 @@ public static void tearDownClass() throws UnknownHostException {
         testOutputFiles.forEach(f -> new File(f).delete());
     }
 
-    @Test
-    public void testGetQuery() throws Exception {
-        List<String> studies = Arrays.asList("s1", "s2");
-        List<String> files = Arrays.asList("f3", "f4", "f5");
-
-        VariantExporterController controller = new VariantExporterController(
-                TestDBRule.getTemporaryDBName(TestDBRule.HUMAN_TEST_DB),
-                                                                             studies, files,
-                                                                             OUTPUT_DIR, evaTestProperties,
-                                                                             emptyFilter);
-
-        // empty query parameters
-        MultivaluedMap<String, String> emptyParameters = new MultivaluedHashMap<>();
-        QueryOptions query = controller.getQuery(emptyParameters);
-        assertEquals(3, query.size());
-        assertEquals(studies, query.getAsStringList(VariantDBAdaptor.STUDIES));
-        assertEquals(files, query.getAsStringList(VariantDBAdaptor.FILES));
-        assertEquals(Arrays.asList("annotation", "sourceEntries.cohortStats"), query.getAsStringList("exclude"));
-
-        // some not accepted parameters
-        MultivaluedMap<String, String> nonAcceptedParameters = new MultivaluedHashMap<>();
-        nonAcceptedParameters.add("wrongParameter1", "1");
-        nonAcceptedParameters.add("wrongParameter1", "1b");
-        nonAcceptedParameters.add("wrongParameter2", "2");
-        query = controller.getQuery(nonAcceptedParameters);
-        assertEquals(3, query.size());
-        assertEquals(studies, query.getAsStringList(VariantDBAdaptor.STUDIES));
-        assertEquals(files, query.getAsStringList(VariantDBAdaptor.FILES));
-        assertEquals(Arrays.asList("annotation", "sourceEntries.cohortStats"), query.getAsStringList("exclude"));
-
-
-        // some accepted parameters
-        MultivaluedMap<String, String> acceptedParameters = new MultivaluedHashMap<>();
```

Comment:
If this code is not needed anymore, we have to be sure that we are still testing at least the same scenarios that before removing Opencga